modules/restart-scanner-container.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * configuration/add-trusted-ca.adoc
+
+:_module-type: PROCEDURE
+[id="restart-scanner_{context}"]
+= Restarting the Scanner container
+
+[role="_abstract"]
+You can restart the Scanner container by deleting the pod.
+
+.Procedure
+
+* Run the following command to delete the Scanner pod:
+** On {ocp}:
++
+[source,terminal]
+----
+$ oc delete po -n stackrox -l app=scanner
```

Comment:
We don't normally use the `po` abbreviation for pod in our docs. Can you update these commands to use `pod` or `pods` instead?